resources/qml/Account/AccountDetails.qml

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Controls 2.1
+
+import UM 1.4 as UM
+import Cura 1.1 as Cura
+
+Column
+{
+    property var profile: null
+    property var loggedIn: false
+
+    padding: 2 * UM.Theme.getSize("default_margin").height
+    spacing: 2 * UM.Theme.getSize("default_margin").height
+
+    AvatarImage
+    {
+        id: avatar
+        width: 75 * screenScaleFactor
+        height: 75 * screenScaleFactor
+        anchors.horizontalCenter: parent.horizontalCenter
+        source: loggedIn ? profile["profile_image_url"] : UM.Theme.getImage("avatar_default")
+        outlineColor: loggedIn ? UM.Theme.getColor("account_widget_ouline_active") : UM.Theme.getColor("account_widget_ouline_inactive")
+    }
+
+    Label
+    {
+        id: message
+        anchors.horizontalCenter: parent.horizontalCenter
+        visible: !loggedIn
+        text: catalog.i18nc("@label", "Please login or create an account to  enjoy all features of Ultimaker Cura")
+    }
+
+    Column
+    {
+        id: userInformation
+        anchors.horizontalCenter: parent.horizontalCenter
+        visible: loggedIn
+
+        Label
+        {
+            anchors.horizontalCenter: parent.horizontalCenter
+            text: loggedIn ? profile["username"] : ""
+            font: UM.Theme.getFont("large")
+        }
+
+        Label
+        {
+            anchors.horizontalCenter: parent.horizontalCenter
+            text: "email.address@hardcoded.is"
```

Comment:
And as long as something is hardcoded (as a fallback?), there's RFC 2606 which describes which domain to use (ie: example.com). The domain hardcoded.is doesn't currently resolve, but it could in the future. So ```email.address@example.com``` would be better.

Reply:
It was just a temporary thing that I forgot behind, because in the current model we don't have the email. I need to add it, but I forgot it.

Reply:
There's nothing as permanent as a temporary solution ;-)

It is a good idea to use best practices even for temporary solutions, because it would not be the first time that a temporary solution makes it into production. Also bad habits die hard, even if you started them with temporary things.

Reply:
Removed in a4c609d514a7687b318613d862894c67dce83b1d since it's not public data and it's not needed at all.